Documentation/ApiOverview/Fal/Administration/Storages.rst

```diff
@@ -53,3 +53,33 @@ Is online?
         within your web root or accessible via a third-party storage service
         which is publicly available. The files will still be available to anyone
         who knows the path to the file.
+
+    ..  versionchanged:: 11.5.35/12.4.11
+
+    Assuming that a web project is located in the directory
+    :file:`/var/www/example.org/` (the "project root path" for Composer-based
+    projects) and the publicly accessible directory is located at
+    :file:`/var/www/example.org/public/` (the "public root path"), accessing
+    resources via the File Abstraction Layer component is limited to the
+    mentioned directories.
+
+    To grant **additional** access to directories, they must be explicitly
+    configured in the system settings of
+    :ref:`$GLOBALS['TYPO3_CONF_VARS']['BE']['lockRootPath'] <typo3ConfVars_be_lockRootPath>`
+    - either using the Install Tool or according to deployment techniques.
+
+    Example:
+
+    ..  code-block:: php
+        :caption: config/system/settings.php
+
+        $GLOBALS['TYPO3_CONF_VARS']['BE']['lockRootPath'] = [
+            ‘/var/shared/documents/’,
+            ‘/var/shared/images/’,
+        ];
+
+    Storages that reference directories not explicitly granted will be marked as
+    "offline" internally - no resources can be used in the website's frontend
+    and backend context.
```

Comment:
Maybe add:

"You cannot use relative directory references (:file:`../../assets/`) or shortcuts like :file:`~/assets`), but only absolute references."

Reply:
You can, I use them for a custom application storage, as the absolute paths are differently on production and development.

Reply:
Oh, ok. I just echoed this from Stefan Frömkens warning in #typo3-cms today:

> Working with sys_file_storage records outside of project- and webroot was possible since years and I have seen some customers using this to share images between two different TYPO3 instances. While installing the current security patches this will not work anymore out of the box. You have to add the full absolute path to your shared folder in [BE][lockRootPath] . Something like ~/  or relative paths like ../../ are not working here. Keep in mind to update this option while moving your TYPO3 instance into another directory or moving to another hoster 

Did you test if your relative storage still works after the security patch?

Reply:
Yup. Worked.

Reply:
I assume, the "~" or "../../" navigated outside the project's root folder.

Reply:
Ah, I was referring to "storage"
Specifically like the term used here for a "lockRootPath storage", not a "FAL storage".

So what I initially meant was that we should note that those lockRootPath directory entries in the array need to be absolute paths, would that be accurate?!

Reply:
Done as comment.